components/config/src/config/mod.rs

```diff
@@ -102,6 +102,8 @@ pub struct Config {
     pub generate_sitemap: bool,
     /// Enables the generation of robots.txt
     pub generate_robots_txt: bool,
+    /// Whether to exclude paginated pages in sitemap; can take values "none", "all"
+    pub exclude_paginated_pages_in_sitemap: String,
```

Comment:
Sorry I meant something like `LinkCheckerLevel` where it's a String for the end user but we do turn it into a Rust enum: https://github.com/getzola/zola/blob/master/components/config/src/config/link_checker.rs#L7-L36

Reply:
You're correct. The main thing is that using an enum with serde serialization you will get error messages if you use invalid string values. Here they will just be ignored which isn't great.

Reply:
Got it, let me make the change right away